.github/workflows/deploy-test-runner.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy the test runner vm to azure
+
+on:
+    workflow_call:
+    workflow_dispatch:
```

Comment:
Though maybe not. Since it's adding time / resources. So maybe just for when merging onto `develop` and/or `main` and as one of the steps as part of the publish to pypi workflow?

Reply:
will do it on the push to develop and main